Microsoft Defender/microsoft-defender-for-endpoints/_meta/smart-descriptions.json

```
@@ -168,5 +200,71 @@
         "field": "host.name"
       }
     ]
+  },
+  {
+    "value": "{service.name} has associated the file {file.name} with the threat: {microsoft.defender.threat.family} (sha1: {file.hash.sha1}).",
+    "conditions": [
+      {
+        "field":"microsoft.defender.entity.type",
+        "value": "File"
+      },
+      {
+        "field":"microsoft.defender.threat.family"
+      },
+      {
+        "field":"file.hash.sha1"
+      },
+      {
+        "field":"file.name"
+      },
+      {
+        "field":"service.name"
+      }
+    ]
+  },
+  {
+    "value": "An alert ({microsoft.defender.alert.id}) has been raised for application {action.properties.Application} ",
```

```suggestion
    "value": "An alert ({microsoft.defender.alert.id}) has been raised for {action.properties.Application} ",
```

I would say `for the application` or `for {application name}`